Common/UnitDefinitions/HeatTransferCoefficient.json

```diff
@@ -33,14 +33,26 @@
       ]
     },
     {
-      "SingularName": "BtuPerSquareFootDegreeFahrenheit",
-      "PluralName": "BtusPerSquareFootDegreeFahrenheit",
+      "SingularName": "BtuPerHourSquareFootDegreeFahrenheit",
+      "PluralName": "BtusPerHourSquareFootDegreeFahrenheit",
       "FromUnitToBaseFunc": "{x} * 5.6782633411134878",
       "FromBaseToUnitFunc": "{x} / 5.6782633411134878",
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "Btu/ft²·hr·°F" ]
+          "Abbreviations": [ "Btu/h·ft²·°F", "Btu/ft²·h·°F", "Btu/hr·ft²·°F", "Btu/ft²·hr·°F" ]
```

Comment:
@angularsen, let me know if this is okay. I fixed the unit name which incorrectly omitted the time as divisor.

Reply:
Adding more abbreviations is fine, but we can't rename the unit as it would be a breaking change.

We would have to 
- Create a copy of the unit with the correct name
- Mark the old one as obsolete by adding `ObsoleteText` property on the old unit in JSON, explaining to use the correctly named unit.
- Remove in next major version, tracked by #1200

Reply:
Thinking more on it, the name is so wrong that we probably should fix it now instead of waiting for v6.

How about following the above steps to obsolete the unit, create a new one, but change its single unit abbreviation to `"Btu/ft²·°F"` WITHOUT the `hr` component. This way we avoid ambiguity on parsing until the unit is removed.

Reply:
Sounds good, I will summit the changes.